src/server.rs

```
@@ -0,0 +1,70 @@
+use actix::{prelude::*, SystemRunner};
+use actix_web::{fs::StaticFiles, server::HttpServer, App};
+
+pub struct Server {
+    system: SystemRunner,
+}
+
+impl Server {
+    /// Contructs a new HTTP server listening on the given address and port.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// let server = Server::new("localhost", "6666");
+    /// assert_eq!(server.is_ok());
+    /// ```
+    pub fn new<S: Into<String>>(address: S, port: S) -> Result<Self, ()> {
```

I would return `failure::Error` as error instead of the empty tuple